src/widgets/widgets-collection.js

```diff
@@ -15,12 +15,12 @@ module.exports = Backbone.Collection.extend({
     this.bind('change:order', _.debounce(this._onChangeOrder.bind(this), 10), this);
     // If a category model applies the category colors, rest should remove/disable
     // the category colors applied before.
-    this.bind('change:isColorsApplied', function (m, isColorCategorized) {
+    this.bind('change:isAutoStyle', function (m, isColorCategorized) {
```

Comment:
I'm a bit confused about the `isAutoStyle` attribute... We're setting [the `autoStyle` attribute in the dataview](https://github.com/CartoDB/deep-insights.js/pull/304/files#diff-9174d99f35e6d799c1c69c256b44b3c3R59), but I don't see where (apart from the tests we are setting `isAutoStyle`)...